### ISSUE TITLE
upgrade_test: 'Failed to load schema version' events aren't filtered out (ignored) as expected

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla-cluster-tests#1628]
### Prerequisites

* [ ] Are you rebased to master ?
* [x] Is it reproducible ?
* [x] Did you perform a cursory search if this issue isn't opened ?

### Versions
* **SCT**: branch-2019.1

### Logs
* test_id: 08dcd064-a0a3-4730-bd22-5a4b0781120f
* job log: https://cloudius-jenkins-test.s3.amazonaws.com/08dcd064-a0a3-4730-bd22-5a4b0781120f/20191226_133117/sct-runner-08dcd064.zip
* test job: https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/rolling-upgrade-2019.1/job/rolling-upgrade-centos7/13/execution/node/38/log/

### Description

'Failed to load schema version' events aren't filtered out (ignored)
We can still see them in critical.log, which causes the job failed.

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

```
[amos@amos-centos8 08dcd064-a0a3-4730-bd22-5a4b0781120f]$ cat sct.log |egrep 'a Node|begin$|ended$'
< t:2019-12-26 12:48:22,388 f:upgrade_test.py l:473  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 begin
< t:2019-12-26 12:48:22,389 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-26 12:52:39,866 f:upgrade_test.py l:475  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 ended
< t:2019-12-26 12:55:59,562 f:upgrade_test.py l:499  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 begin
< t:2019-12-26 12:55:59,562 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-26 12:59:52,042 f:upgrade_test.py l:501  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 ended
< t:2019-12-26 13:02:47,648 f:upgrade_test.py l:516  c:sdcm.tester          p:INFO  > Rollback Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 begin
< t:2019-12-26 13:02:47,649 f:upgrade_test.py l:207  c:sdcm.tester          p:INFO  > Rollbacking a Node
< t:2019-12-26 13:05:00,722 f:upgrade_test.py l:518  c:sdcm.tester          p:INFO  > Rollback Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 ended
< t:2019-12-26 13:07:27,908 f:upgrade_test.py l:525  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 begin
< t:2019-12-26 13:07:27,908 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-26 13:11:21,822 f:upgrade_test.py l:527  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-4 ended
< t:2019-12-26 13:13:16,882 f:upgrade_test.py l:525  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-1 begin
< t:2019-12-26 13:13:16,883 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-26 13:17:04,917 f:upgrade_test.py l:527  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-1 ended
< t:2019-12-26 13:18:58,837 f:upgrade_test.py l:525  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-3 begin
< t:2019-12-26 13:18:58,837 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-26 13:22:43,630 f:upgrade_test.py l:527  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-3 ended
```

```
[amos@amos-centos8 08dcd064-a0a3-4730-bd22-5a4b0781120f]$ cat critical.log 
2019-12-26 12:51:43.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=3627 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:51:43+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.14: std::runtime_error (Failed to load schema version 51884e0d-dc57-3119-a9a8-69d36ba653f3)
2019-12-26 12:51:43.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=3627 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:51:43+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.14: std::runtime_error (Failed to load schema version 51884e0d-dc57-3119-a9a8-69d36ba653f3)
2019-12-26 12:51:43.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=3627 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:51:43+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.14: std::runtime_error (Failed to load schema version 51884e0d-dc57-3119-a9a8-69d36ba653f3)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=4078 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=4078 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=4078 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=4079 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=4079 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:46.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=4079 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:46+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=4083 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=4083 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=4083 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=4084 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=4084 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:52:55.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=4084 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:52:55+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 1] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
<UnknownTimestamp>: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=5135 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
<UnknownTimestamp>: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=5135 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
<UnknownTimestamp>: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=5135 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:55:06.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=5136 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:55:06.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=5136 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 12:55:06.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=5136 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T12:55:06+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 3] storage_proxy - Exception when communicating with 10.142.0.67: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:11:32.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=7389 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:11:32+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:11:32.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=7389 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:11:32+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:11:32.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=7389 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:11:32+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:12:34.000: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=8057 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:12:34+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:12:34.000: (DatabaseLogEvent Severity.CRITICAL): type=SCHEMA_FAILURE regex=Failed to load schema version line_number=8057 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:12:34+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
2019-12-26 13:12:34.000: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=8057 node=Node rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 [35.185.35.39 | 10.142.0.61] (seed: False)
2019-12-26T13:12:34+00:00  rolling-upgrade-upgrade--centos-db-node-08dcd064-0-2 !ERR     | scylla: [shard 0] storage_proxy - Exception when communicating with 10.142.0.63: std::runtime_error (Failed to load schema version 878cb162-fb67-3078-9686-05b590cb709e)
```

/CC @fruch

**Comment by amoskong** [comment]
I'm testing this fix:

```
commit bd7e47238d1079f6bfd325f62ec3c0f8ee17d399
Author: Amos Kong <amos@scylladb.com>
Date:   Fri Dec 27 03:29:11 2019 +0800

    fix event filter: Failed to load schema
    
    Signed-off-by: Amos Kong <amos@scylladb.com>

diff --git a/upgrade_test.py b/upgrade_test.py
index 7fea5437..c2395bd8 100644
--- a/upgrade_test.py
+++ b/upgrade_test.py
@@ -415,9 +415,9 @@ class UpgradeTest(FillDatabaseData):
         well, upgrade all nodes to new version in the end.
         """
 
-        filter_errors = [{'line': 'Failed to load schema', 'type': 'DATABASE_ERROR'},
-                         {'line': 'Failed to load schema', 'type': 'SCHEMA_FAILURE'},
-                         {'line': 'Failed to pull schema', 'type': 'DATABASE_ERROR'},
+        filter_errors = [{'line': 'Failed to load schema version', 'type': 'DATABASE_ERROR'},
+                         {'line': 'Failed to load schema version', 'type': 'SCHEMA_FAILURE'},
+                         {'line': 'Failed to pull schema version', 'type': 'DATABASE_ERROR'},
                          {'line': 'Backtrace:', 'type': 'BACKTRACE'}]
 
         for error in filter_errors:
```

**Comment by amoskong** [comment]
@fruch I didn't exactly know the principle. If we create the DbEventsFilter objects one by one, only the last one really affects.
If we create the DbEventsFilter objects _once_  (the statement is connected by `,`), all filters work as expected.

#### A good example:

```
        with DbEventsFilter(type='RUNTIME_ERROR', line='Failed to load schema'), DbEventsFilter(type='SCHEMA_FAILURE', line='Failed to load schema'):
            DatabaseLogEvent(type="RUNTIME_ERROR", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                                   line=known_failure_line)
            DatabaseLogEvent(type="SCHEMA_FAILURE", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                                   line=known_failure_line)
            DatabaseLogEvent(type="NO_SPACE_ERROR", regex="B").add_info_and_publish(node="B", line_number=22,
                                                                                    line="not filtered")
```

#### A bad example:  (only SCHEMA_FAILURE event will be filtered out)
```
        with DbEventsFilter(type='RUNTIME_ERROR', line='Failed to load schema'), DbEventsFilter(type='SCHEMA_FAILURE', line='Failed to load schema'):
            DatabaseLogEvent(type="RUNTIME_ERROR", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                                   line=known_failure_line)
            DatabaseLogEvent(type="SCHEMA_FAILURE", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                                   line=known_failure_line)
            DatabaseLogEvent(type="NO_SPACE_ERROR", regex="B").add_info_and_publish(node="B", line_number=22,
                                                                                    line="not filtered")
```

#### A Good example:
```
        DbEventsFilter(type='RUNTIME_ERROR', line='Failed to load schema')
        DbEventsFilter(type='SCHEMA_FAILURE', line='Failed to load schema')
        DatabaseLogEvent(type="RUNTIME_ERROR", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                               line=known_failure_line)
        DatabaseLogEvent(type="SCHEMA_FAILURE", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                               line=known_failure_line)
        DatabaseLogEvent(type="NO_SPACE_ERROR", regex="B").add_info_and_publish(node="B", line_number=22,
                                                                                line="not filtered")
```

#### A bad example:
```
        DbEventsFilter(type='RUNTIME_ERROR', line='Failed to load schema')
        DbEventsFilter(type='SCHEMA_FAILURE', line='Failed to load schema')
        DatabaseLogEvent(type="RUNTIME_ERROR", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                               line=known_failure_line)
        DatabaseLogEvent(type="SCHEMA_FAILURE", regex="B").add_info_and_publish(node="A", line_number=22,
                                                                               line=known_failure_line)
        DatabaseLogEvent(type="NO_SPACE_ERROR", regex="B").add_info_and_publish(node="B", line_number=22,
                                                                                line="not filtered")
```

**Comment by amoskong** [comment]
The issue is fixed @roydahan in by https://github.com/scylladb/scylla-cluster-tests/pull/1629/files  (reviewing state)

**Comment by amoskong** [comment]
Update:

This issue had been fixed on master, and I just posted a PR to fix the issue on branch-3.2
